Week11/app/src/main/java/com/example/marvelcharacters/view/characterdetail/CharacterDetailActivity.kt

```diff
@@ -0,0 +1,60 @@
+package com.example.marvelcharacters.view.characterdetail
+
+import android.content.Context
+import android.content.Intent
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import com.bumptech.glide.Glide
+import com.example.marvelcharacters.R
+import com.example.marvelcharacters.model.entities.Character
+import com.example.marvelcharacters.model.entities.Thumbnail
+import kotlinx.android.synthetic.main.activity_character_detail.*
+import kotlinx.android.synthetic.main.character_view_holder.view.*
+
+fun Context.characterDetailActivityIntent(
+    characterName: String,
+    characterDescription: String,
+    characterThumbnail: Thumbnail
+): Intent {
```

Comment:
I don't think creating an extension function like this makes sense because it's not generally used. It's only for one use-case.
It's better to create a companion object in DetailActivity called `getIntent()` or something similar. Then you would just call `startActivity(DetailActivity.getIntent)`